src/views/index.hbs

```diff
@@ -6,11 +6,169 @@
 			</h1>
 		</div>
 
+
+    	<!--charts-->
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.4/Chart.js"></script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/chartjs-plugin-datalabels@0.4.0/dist/chartjs-plugin-datalabels.min.js"></script> 
+    
+
+
+    <!--pie chart-->
+  
+     <div style="margin-left:5%;margin-right:5%">
+        <canvas id="myChart" style="width:100%;max-width:80%;"></canvas>
+    </div>
+
+  <script>
```

Comment:
Your scripts should be defined in a separate js file and imported here via `<script ... url=your_script_file >`